plugins/brew/README.md

```diff
@@ -10,17 +10,24 @@ plugins=(... brew)
-| `bcubo`  | `brew update && brew outdated --cask` | Update Homebrew data, then list outdated casks.                     |
-| `bcubc`  | `brew upgrade --cask && brew cleanup` | Update outdated casks, then run cleanup.                            |
+| Alias        | Command                               | Description                                                                               |
+|--------------|---------------------------------------|-------------------------------------------------------------------------------------------|
+| `brewp`      | `brew pin`                            | Pin a specified formula so that it's not upgraded.                                        |
+| `brews`      | `brew list -1`                        | List installed formulae or the installed files for a given formula.                       |
+| `brewsp`     | `brew list --pinned`                  | List pinned formulae, or show the version of a given formula.                             |
+| `bubo`       | `brew update && brew outdated`        | Update Homebrew data, then list outdated formulae and casks.                              |
+| `bubc`       | `brew upgrade && brew cleanup`        | Upgrade outdated formulae and casks, then run cleanup.                                    |
+| `bubu`       | `bubo && bubc`                        | Do the last two operations above.                                                         |
+| `buf`        | `brew upgrade --formula`              | Upgrade only formulas (not casks).                                                        |
+| `bcubo`      | `brew update && brew outdated --cask` | Update Homebrew data, then list outdated casks.                                           |
+| `bcubc`      | `brew upgrade --cask && brew cleanup` | Update outdated casks, then run cleanup.                                                  |
+| `bsl`        | `brew services list`                  | List all running services for the current user (or root).                                 |
+| `bsr`        | `brew services run`                   | Run the service formula without registering to launch at login (or boot).                 |
+| `bson`       | `brew services start`                 | Start the service formula immediately and register it to launch at login (or boot).       |
+| `bsoff`      | `brew services stop`                  | Stop the service formula immediately and unregister it from launching at login (or boot). |
+| `bsrunall`   | Run `bsr` for each service            | Run all stopped services.                                                                 |       
+| `bsstartall` | Run `bson` for each service           | Start all stopped services.                                                               |
+| `vsstopall`  | Run `bsoff` for each service          | Stop all started services.                                                                |
```

Comment:
@tinogomes Thanks for the contribution, would you be able to modify this to update the README (think there is a typo) and rebase with the latest homebrew plugin aliases?

Reply:
@robbyrussell done!